### PR TITLE
fix event

### PR DESCRIPTION
**Pull request by mikeweiwei**
Add more event.When node is deleted and then add envet

### CODE REVIEW

File: cluster-autoscaler/core/utils.go [hunk 1]
```diff
@@ -404,13 +404,15 @@ func removeOldUnregisteredNodes(unregisteredNodes []clusterstate.UnregisteredNod
 				glog.Warningf("Failed to remove node %s: node group min size reached, skipping unregistered node removal", unregisteredNode.Node.Name)
 				continue
 			}
-			logRecorder.Eventf(apiv1.EventTypeNormal, "DeleteUnregistered",
-				"Removing unregistered node %v", unregisteredNode.Node.Name)
 			err = nodeGroup.DeleteNodes([]*apiv1.Node{unregisteredNode.Node})
 			if err != nil {
 				glog.Warningf("Failed to remove node %s: %v", unregisteredNode.Node.Name, err)
 				return removedAny, err
+				logRecorder.Eventf(apiv1.EventTypeNormal, "DeleteUnregisteredFailed",
```
Comment (feiskyer) [inline cluster-autoscaler/core/utils.go]: move eventf above return?

**Comment by MaciekPytel** [comment]
Please squash your changes into a single commit.

**Comment by mwielgus** [comment]
I squashed everything in github.

**Pull request merged**